webui/tests/cypress/support/index.js

```diff
@@ -15,4 +15,10 @@
 
 import './commands';
 
-Cypress.Cookies.defaults({ whitelist: /auth/ });
+// window.localStorage.setItem('global/server-address', JSON.stringify('http://localhost:8081'));
+// const _clear = Cypress.LocalStorage.clear;
```

Comment:
another approach to whitelisting local storage.
TODO remove

Reply:
if we could only bypass cleaning a single key this could be viable but I don't how we could without monkey patching